src/components/Input/Input.stories.tsx

```diff
@@ -172,52 +172,3 @@ export const inputAttributes = () => (
     </HelperErrorText>
   </>
 );
-
-export const RadioInput = () => {
-  const [stateValue, setStateValue] = useState("");
```

Comment:
Minor: since this was removed `useState` can be removed from the import. Github gives a nice message about it:

<img width="705" alt="Screen Shot 2021-04-09 at 11 41 09 AM" src="https://user-images.githubusercontent.com/1280564/114205726-9c99d180-9928-11eb-8f91-86a95080f8d4.png">

Reply:
Looks like the Github Action test failed because of this linting issue as well.

Reply:
Fixed!  Thanks for catching that!